geoutils/georaster/raster.py

```diff
@@ -2331,19 +2318,19 @@ def interp_points(
         #         rpts.append(zint)
         # rpts = np.array(rpts)
 
-    def split_bands(self: RasterType, copy: bool = False, subset: list[int] | int | None = None) -> list[Raster]:
+    def split_indexes(self: RasterType, copy: bool = False, subset: list[int] | int | None = None) -> list[Raster]:
```

Comment:
I think in this case, it makes more sense to keep "band" as it is much clearer what this will do. To me, indexes refer rather to the position of the bands, so it does not make sense to split them.

Reply:
Agree, changed